packages/front-end/pages/fact-metrics/[fmid].tsx

```diff
@@ -185,6 +197,22 @@ export default function FactMetricPage() {
   const [editTagsModal, setEditTagsModal] = useState(false);
   const [editOwnerModal, setEditOwnerModal] = useState(false);
 
+  const [tab, setTab] = useLocalStorage<MetricTab>(
+    `metricTabbedPageTab__${fmid}`,
+    "analysis"
+  );
+  useEffect(() => {
+    const handler = () => {
+      const hash = window.location.hash.replace(/^#/, "") as MetricTab;
+      if (metricTabs.includes(hash)) {
+        setTab(hash);
+      }
+    };
+    handler();
+    window.addEventListener("hashchange", handler, false);
+    return () => window.removeEventListener("hashchange", handler, false);
+  }, [setTab]);
```

Comment:
We have a `Tabs` component that already includes this logic.  We should use that instead.